python/examples/experimental/custom_rank_metrics.ipynb

```diff
@@ -0,0 +1,242 @@
+    "    data=single_df,\n",
+    "    prediction_column=\"raw_predictions\",\n",
+    "    target_column=\"raw_targets\",\n",
+    "    log_full_data=True\n",
+    ")\n",
+    "\n",
+    "# NOTE: If you've already ran why.log() on your input data, change log_full_data to False."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# View results\n",
+    "results.view().to_pandas().T"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Send profile summaries to WhyLabs platform\n",
+    "Required information can be found at https://hub.whylabsapp.com under the Settings > Model and Dataset Management page.\n",
+    "\n",
+    "API keys are only shown once, so you may need to create a new one and save somewhere safe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure WhyLabs info, if needed\n",
+    "import getpass\n",
+    "import os\n",
+    "\n",
+    "os.environ[\"WHYLABS_DEFAULT_ORG_ID\"] = input(\"Enter your WhyLabs Org ID\")\n",
+    "os.environ[\"WHYLABS_DEFAULT_DATASET_ID\"] = input(\"Enter your WhyLabs Dataset ID\")\n",
+    "os.environ[\"WHYLABS_API_KEY\"] =  getpass.getpass(\"Enter your WhyLabs API key\")\n",
+    "print(\"Using API Key ID: \", os.environ[\"WHYLABS_API_KEY\"][0:10])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write the results to WhyLabs platform\n",
+    "results.writer(\"whylabs\").write()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## WhyLabs UI Improvements\n",
+    "\n",
+    "### Optional: Move to Outputs tab\n",
+    "The above metrics will default to the Input tab, but you may organize your data columns by instead displaying relevant columns in the Outputs tab.\n",
+    "\n",
+    "There are two options to do so:\n",
+    "1. Using WhyLabsWriter API within whylogs\n",
+    "2. Using REST API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Update relevant columns to output columns\n",
+    "from whylogs.api.writer.whylabs import WhyLabsWriter\n",
+    "k = 2 \n",
+    "column_names = [\"mean_average_precision_k_\"+str(k), \"accuracy_k_\"+str(k), \n",
+    "                \"mean_reciprocal_rank\", \"precision_k_\"+str(k), \"recall_k_\"+str(k), \n",
+    "                \"top_rank\", \"average_precision_k_\"+str(k)]\n",
+    "\n",
+    "whylabs_writer = WhyLabsWriter()\n",
+    "try:\n",
+    "    whylabs_writer.tag_output_columns(column_names)\n",
+    "except:\n",
+    "    print(\"Issue with changing UI via whylogs. \\nUse REST API at \"\n",
+    "          \"https://api.whylabsapp.com/swagger-ui#/Models/PutEntitySchemaColumn.\")"
```

Comment:
I don't think we should guide people to the REST API directly, can we fix the tag_output_columns if there is an error?